tests/suite/model/cite.typ

```diff
@@ -171,3 +171,72 @@ aaa
 // Test warning for deprecated alias.
 // Warning: 18-37 style "chicago-fullnotes" has been deprecated in favor of "chicago-notes"
 #set cite(style: "chicago-fullnotes")
+
+--- cite-supplements-and-ibid ---
+#set page(width: 300pt)
+
+Par 1 @arrgh
+
+Par 2 @arrgh[p. 5-8]
+
+Par 3 @arrgh[p. 5-8]
+
+Par 4 @arrgh[p. 9-10]
```

Comment:
Let's add a quick showcase of how it behaves in contrast with the plain_text alternative: the strong should be considered a different supplement.

FWIW, I think this is the best approach for the time being. We can consider changing it in the future if necessary.

```suggestion

Par 1 @arrgh

Par 2 @arrgh[p. 5-8]

Par 3 @arrgh[p. 5-8]

Par 4 @arrgh[p. 9-10]

Par 5 @arrgh[**p. 9-10**]
```